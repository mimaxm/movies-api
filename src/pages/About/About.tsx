import { FC } from "react";
import { Header } from "../../components/Header/Header";
import movieImg from "./movie.png";

export const About: FC = () => {
	return (
		<>
			<Header />
			<div className="container">
				<h2>About page</h2>
				<div className="center">
					<img src={movieImg} alt="about" />
				</div>
				<p>
					Lorem ipsum dolor sit, amet consectetur adipisicing elit. Nostrum
					eaque vel repellendus recusandae exercitationem quos totam, neque,
					nemo beatae aut, est harum laudantium aliquam aspernatur molestias
					officia pariatur nesciunt corporis.
				</p>
				<p>
					Ut voluptas sit, optio dolor ullam ipsa amet dolorum labore quisquam
					vero ab modi animi dignissimos tenetur corporis pariatur
					reprehenderit. Illo qui sunt error aperiam eveniet expedita ipsum aut
					aspernatur.
				</p>
				<p>
					Quasi porro laudantium quidem commodi numquam delectus cupiditate ipsa
					tempora consequuntur molestias distinctio mollitia animi quibusdam
					architecto, excepturi est ea sit dolores nobis labore a maxime
					perspiciatis doloribus tenetur! Ea.
				</p>
			</div>
		</>
	);
};
