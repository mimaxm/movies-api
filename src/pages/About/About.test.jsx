import React from 'react';
import { About } from './About';
import { shallow } from 'enzyme';

describe('About page testing', () => {
   const component = shallow(<About />);
   test('component should render', () => {
      expect(component).toMatchSnapshot();
   });
   test('component should include 3 "p"', () => {
      expect(component.find('p')).toHaveLength(3);
   });
})