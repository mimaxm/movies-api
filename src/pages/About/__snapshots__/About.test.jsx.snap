// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About page testing component should render 1`] = `
<Fragment>
  <Header />
  <div
    className="container"
  >
    <h2>
      About page
    </h2>
    <div
      className="center"
    >
      <img
        alt="about"
        src={
          Object {
            "process": [Function],
          }
        }
      />
    </div>
    <p>
      Lorem ipsum dolor sit, amet consectetur adipisicing elit. Nostrum eaque vel repellendus recusandae exercitationem quos totam, neque, nemo beatae aut, est harum laudantium aliquam aspernatur molestias officia pariatur nesciunt corporis.
    </p>
    <p>
      Ut voluptas sit, optio dolor ullam ipsa amet dolorum labore quisquam vero ab modi animi dignissimos tenetur corporis pariatur reprehenderit. Illo qui sunt error aperiam eveniet expedita ipsum aut aspernatur.
    </p>
    <p>
      Quasi porro laudantium quidem commodi numquam delectus cupiditate ipsa tempora consequuntur molestias distinctio mollitia animi quibusdam architecto, excepturi est ea sit dolores nobis labore a maxime perspiciatis doloribus tenetur! Ea.
    </p>
  </div>
</Fragment>
`;
